Src/DependencyCollector/Net46.Tests/DependencyTrackingTelemetryModuleTestNet46.cs

```diff
@@ -74,6 +86,9 @@ public void TestDependencyCollectionNoParentActivity()
                     -TimeSpan.FromMinutes(1).TotalMilliseconds, 
                     "now - 1 min < timestamp");
                 Assert.AreEqual("200", item.ResultCode);
+                Assert.AreEqual(
+                    SdkVersionHelper.GetExpectedSdkVersion(typeof(DependencyTrackingTelemetryModule), "rdddsd:"),
```

Comment:
never mind - i didn't see you are setting EnableDiagnosticSourceInstrumentation = true at the beginning.